core/git_command.py

```diff
@@ -279,9 +279,11 @@ def git(
                 raise GitSavvyError(str(e), show_panel=show_panel_on_error, window=window)
 
         stdout, stderr = None, None
+        savvy_env = self.savvy_settings.get("env")
+        savvy_env_expanded = {k: sublime.expand_variables(v, os.environ) for k, v in savvy_env.items()}
```

Comment:
At this point `savvy_env` can be `None`.  That's why we do the `or {}` in the original code.

Reply:
I document feature and fix the None issue